release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -298,6 +298,89 @@ With this release, the `MaxUnavailableStatefulSet` featureset configuration para
 [id="ocp-4-14-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-14-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+This release includes the following version updates for monitoring stack components and dependencies:
+
+* kube-state-metrics to 2.9.2
+* node-exporter to 1.6.1
+* prom-label-proxy to 0.7.0
+* Prometheus to 2.46.0
+* prometheus-operator to 0.67.1
+
+[id="ocp-4-14-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* *New*
+** Added the `NodeSystemSaturation` alert to monitor resource saturation on a node.
+** Added the `NodeSystemdServiceFailed` alert to monitor the systemd service on a node.
+** Added the `NodeMemoryMajorPagesFaults` alert to monitor major page faults on a node.
+** Added the `PrometheusSDRefreshFailure` alert to monitor failed Prometheus service discoveries.
```

Comment:
KubeDeploymentRolloutStuck is added since 4.14,
https://github.com/openshift/cluster-monitoring-operator/blob/release-4.14/assets/control-plane/prometheus-rule.yaml#L53
no such alert in 4.13
https://github.com/openshift/cluster-monitoring-operator/blob/release-4.13/assets/control-plane/prometheus-rule.yaml

Reply:
@jan--f - Should we mention dropped alerts? I don't think we've done so in the past, but I think it might be good to let users know what has been dropped. WDYT?

Reply:
Added `KubeDeploymentRolloutStuck` to the list

Reply:
If it was not an alert that would fire in a customer cluster, it's safe to leave it out here, I think.